habitat_baselines/config/pointnav/ppo_pointnav.yaml

```diff
@@ -1,26 +1,35 @@
-BASE_TASK_CONFIG_PATH: "configs/tasks/pointnav.yaml"
+# Note:  Hyperparameters have been changed slightly from
+# the paper to allow for things to easily run on 1 GPU
+
+BASE_TASK_CONFIG_PATH: "configs/tasks/pointnav_gibson.yaml"
 TRAINER_NAME: "ppo"
 ENV_NAME: "NavRLEnv"
 SIMULATOR_GPU_ID: 0
 TORCH_GPU_ID: 0
-VIDEO_OPTION: ["disk", "tensorboard"]
+# VIDEO_OPTION: ["disk", "tensorboard"]
```

Comment:
Please, remove or leave it uncommented.

Reply:
I left it there for easy video generation for people, but having it be the default when there are 994 episodes to evaluate on is not great :)

Reply:
Let's add comment, something like "Enable video generation for debugging purposes."